src/serializer/ResidualHeapSerializer.js

```diff
@@ -1842,7 +1842,7 @@ export class ResidualHeapSerializer {
           : t.callExpression(functionExpression, []);
         ast_body.push(t.expressionStatement(callExpression));
       } else {
-        ast_body = body;
+        Array.prototype.push.apply(ast_body, body);
```

Comment:
It is not obvious to me why this is not jut "ast_body.push(body)".

Reply:
Because body is an array of elements, and not just an element.